packages/ui/src/components/SelectInputV2/SelectBar.tsx

```diff
@@ -451,3 +565,5 @@ export const SelectBar = ({
     </Tooltip>
   )
 }
+
+export { SelectBar, StyledInputWrapper }
```

Comment:
Another issue here also, there is space but somehow it only shows 2 tags fully
![Screenshot 2025-05-27 at 10 33 02](https://github.com/user-attachments/assets/b47db7cd-0c33-48b5-be15-55561bea42a1)

Reply:
![Screenshot 2025-05-27 at 10 33 51](https://github.com/user-attachments/assets/fe7bf7b0-51f3-4ee4-a98d-b14a62aa8665)

sometimes it also show them big somehow